extensions/amp-story/amp-story-animation.md

```diff
@@ -0,0 +1,648 @@
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-story-animation
+
+The `<amp-story-animation>` element defines a custom animation effect in an
+[`amp-story`](./amp-story.md). These effects rely on the [Web Animations API](https://www.w3.org/TR/web-animations/) to define and run animations in AMP Stories.
+
+## Example
+
+```html
+<amp-story-page id="cover">
+  <amp-story-grid-layer template="fill">
+    <div class="zoom">Zoom!</div>
+  </amp-story-grid-layer>
+  <amp-story-animation layout="nodisplay" trigger="visibility">
+    <script type="application/json">
+      {
+        "selector": ".zoom",
+        "iterations": "infinite",
+        "keyframes": {
+          "transform": ["scale(1)", "scale(1.2)", "scale(1)"]
+        }
+      }
+    </script>
+  </amp-story-animation>
+</amp-story-page>
+```
+
+## Attributes
+
+### trigger
+
+Determines when the animation should be triggered. This must be set to
+`visibility` so the animation starts when a story page becomes visible and active.
+
+### layout
+
+Should always be `nodisplay`.
+
+### id [optional]
+
+An ID can be used to determine [sequencing](./amp-story.md#sequencing-animations)
+when defining multiple animations. Other animated elements or `<amp-story-animation>` effects can use this ID to determine that they should
+start animating only after this effect has finished.
+
+### animate-in-after [optional]
+
+Use this attribute to chain or sequence animations. Specify the ID of the animated element that this element's animation will follow. The element must be present on the same `<amp-story-page>`. For further details, see the [Sequencing animations](./amp-story.md#sequencing-animations) section.
+
+For example, in the following code, the `amp-story-animation` effect starts
+after `object1` completes its entrance:
+
+```html
+<amp-story-page id="page1">
+  <amp-story-grid-layer template="vertical">
+    <div id="object1" animate-in="rotate-in-left">
+      1
+    </div>
+    <div class="custom-animated">
+      2
+    </div>
+  </amp-story-grid-layer>
+  <!-- will start after object1 has finished -->
+  <amp-story-animation
+    layout="nodisplay"
+    trigger="visibility"
+    animate-in-after="object1"
+  >
+    <script type="application/json">
+      {
+        "selector": ".custom-animated",
+        "keyframes": {
+          "opacity": [0, 1],
+          "transform": ["scale(0.5)", "scale(1)"]
+        }
+      }
+    </script>
+  </amp-story-animation>
+</amp-story-page>
+```
+
+## Format
```

Comment:
While this component should have all the relevant information and not link out to `amp-animation`, is it possible to reorder it to match the template a bit better?
https://github.com/ampproject/amphtml/blob/master/build-system/tasks/extension-generator/extension-doc.template.md

Does it have any actions or events it reveals? If not, it may be a special case like `amp-bind`. Happy to talk more about it.